PWGJE/Core/JetDerivedDataUtilities.h

```diff
@@ -341,6 +344,9 @@ uint8_t setTrackSelectionBit(T const& track)
   if (track.isGlobalTrackWoPtEta()) {
     SETBIT(bit, JTrackSel::globalTrack);
   }
+  if (track.isGlobalTrackWoDCA()) {
```

Comment:
Hi Hadi,

Do you know if this track selection applies the pT and eta cut already in the track selection task? We should try to avoid this so we can set them ourselves in the jet finder